base/applications/winver/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 
-add_executable(winver winver.c winver.rc)
+list(APPEND SOURCE
+    osinfo.c
+    winver.c
+    winver_p.h)
```

Comment:
what about more simply "winver.h" ?

Reply:
`<winver.h>` already exists in standard headers.

Reply:
The file name implies "WinVer Program" to avoid the collision with `winver.h` from the SDK.